### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by thechosenjuan**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ const store = configureStore();
 
 ReactDOM.render(
   <Provider store={store}>
-    <LanguageProvider messages={messages}>
-      <App />
-    </LanguageProvider>
+          <LanguageProvider messages={messages}>
+              <App />
+          </LanguageProvider>
   </Provider>,
```
Comment (sbaidon) [inline src/index.js]: What happened on this file? This shouldn't have changed

File: src/router/routes.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import { Route, Switch, withRouter } from "react-router-dom";
+import App from "../views/App";
+import Login from "../views/Login/view";
+
+export const Routes = () => (
+  <Switch>
+    <Route exact path="/" component={App} />
+    <Route exact path="/login" component={Login} />
+  </Switch>
+);
```
Comment (sbaidon) [inline src/router/routes.js]: Where is this file being used? Again I think we have an issue with routers

**Comment by sbaidon** [comment]
Only thing missing is the prettier config, we need to add the `.prettierrc` file

**Comment by thechosenjuan** [comment]
Do you have an example of the prettier conf file?

**Comment by sbaidon** [comment]
https://prettier.io/docs/en/configuration.html

**Comment by thechosenjuan** [comment]
Which configurations are the ones we are going to use?

**Comment by sbaidon** [comment]
Up to you, I would just say lets leave print width at 80, I don't really care about the other ones

**Pull request merged**